cnxauthoring/utils.py

```diff
@@ -43,6 +43,31 @@
     'Translator': 'translators',
     }
 
+NAMESPACES = ('<body xmlns="http://www.w3.org/1999/xhtml" '
+              'xmlns:bib="http://bibtexml.sf.net/" '
+              'xmlns:data="http://dev.w3.org/html5/spec/#custom" '
+              'xmlns:epub="http://www.idpf.org/2007/ops" '
+              'xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" '
+              'xmlns:dc="http://purl.org/dc/elements/1.1/" '
+              'xmlns:lrmi="http://lrmi.net/the-specification">'
+              '{}'
+              '</body>')
+
+
+def manage_namespace(content):
+    try:
+        e = etree.fromstring(content)
+    except etree.XMLSyntaxError:
+        ##################################################
+        # trying to catch error by wrapping content in a #
+        # namespaced body tag                            #
+        ##################################################
```

Comment:
A comment like:
Try and  handle two cases: no root tag (wrap in a <body>) and duplicate xmlns= (upstream error in mathml - not sure if it's mathjax or mathjax/aloha plugin) Note the "big hammer" of recover for that second case.
BBB remove recover once upstream dup xmlns is solved